sound/smooth_swing_v2.h

```diff
@@ -148,6 +166,7 @@ class SmoothSwingV2 : public SaberBasePassThrough {
 	    // We need to stop setting the volume when off, or playback may never stop.
 	    A.set_volume(mixhum * mixab);
 	    B.set_volume(mixhum * (1.0 - mixab));
+      mixhum = delegate_->SetSwingVolume(swing_strength, mixhum);
```

Comment:
fix indentation?

Reply:
are you sure you don't want to do this *before* A.set_volume()
That will duck all the hum volumes, not just the "main" hum.

Reply:
good point, I've moved it.

Reply:
removed the mix hum setting.  the ducking wasn't required.